content/tutorials/r_citations_markdown/index.Rmd

```diff
@@ -190,20 +190,19 @@ The defaults are OK.
 
 ![](global-options.png)
 
-# Citing documents {#citing-documents}
-
+# Citing documents {\#citing-documents}
```

Comment:
I suggest to revert this change, it doesn't follow the recommended markdown syntax.